guacamole/src/main/webapp/app/manage/templates/manageConnection.html

```diff
@@ -23,35 +23,35 @@
 <div class="view" ng-class="{loading: !isLoaded()}">
 
     <div class="logout-panel">
-        <a class="back button" href="#/manage/">{{'MANAGE_CONNECTION.ACTION_NAVIGATE_BACK' | translate}}</a>
-        <a class="logout button" ng-click="logout()">{{'MANAGE_CONNECTION.ACTION_LOGOUT' | translate}}</a>
+        <a class="back button" href="#/manage/">{{'MANAGE_CONNECTION.ACTION_NAVIGATE_BACK'| translate}}</a>
+        <a class="logout button" ng-click="logout()">{{'MANAGE_CONNECTION.ACTION_LOGOUT'| translate}}</a>
     </div>
 
     <!-- Main property editor -->
-    <h2>{{'MANAGE_CONNECTION.SECTION_HEADER_EDIT_CONNECTION' | translate}}</h2>
+    <h2>{{'MANAGE_CONNECTION.SECTION_HEADER_EDIT_CONNECTION'| translate}}</h2>
     <div class="section">
         <table class="properties">
-            
+
             <!-- Edit connection name -->
             <tr>
-                <th>{{'MANAGE_CONNECTION.FIELD_HEADER_NAME' | translate}}</th>
-              
+                <th>{{'MANAGE_CONNECTION.FIELD_HEADER_NAME'| translate}}</th>
```

Comment:
Ok, seriously, I think you must have hit "reformat" in netbeans... We'd better revert this.